### DISCUSSION TITLE
Can I specify multiple upstreams for the same domains

### DISCUSSION THREAD

**Discussion by Jefffish09** [AdguardTeam/AdGuardHome#4528] [Q&A] [answered]
According to this [wiki - Specifying Upstreams For Domains](https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#upstreams-for-domains), we know we can specify **a upstream** for some domains like below:
```
[/google.com/facebook.com/]https://dns.google/dns-query
```
But one question, if I want to specify more than one upstream for these domains, for example, it will parallel queries to **google**, **cloudflare** and **quad9** DNS when trying to resolve "google.com" and "facebook.com". Can I write the configuration like below?
```
[/google.com/facebook.com/]https://dns.google/dns-query
[/google.com/facebook.com/]https://dns.cloudflare.com/dns-query
[/google.com/facebook.com/]https://dns.quad9.net/dns-query
```

**Comment by ainar-g** [comment, marked as answer]
Yes, that should work.
